v0/18-4-0_Custom_work_and_other_record_tabs_and_fields.markdown

```diff
@@ -1,8 +1,123 @@
+
+__Important__ You must always add a field to a tab, to ensure it remains editable and visible in System Administration.  If you do not, you'll have an empty tab in your record which you will be unable to edit.  In this case, please contact Lamplight for support.
+
+![System Admin - tab information]({{imgpath}}1219a.png)
+
+__Editing a tab__
+
+To edit a tab, select the tab in the left-hand column, and when the tab information shows in the middle column, double click on the text 'Tab name', and edit the existing information (as above)
+
+__Creating a field__
+
+Select the required tab, and in the middle column, choose the button 'Click to add'. In the right-hand column, you'll see a new section appear 'Field: Click to add.'  Double click on the non-bold text 'Field: Click to add ‘and a pop--up box will open.  Again, some of the options show here are redundant for work records.
+
+![System Admin - field information]({{imgpath}}1220a.png)
+
+* Text: This is the name of the field.
+* What type of person is this for:  No answer can be selected here, as records show all fields for all users.
+
+* Type: This selects the type of field you will be creating.  By default, a new field will be a 'select' field, unless changed.  See information on types of fields below.
+
+* Is this a required for: IF this is selected, a user will not be able to exit the tab without entering information.  Use very cautiously.
+* Restrict this field to managers/admins?:  Although you can tick this field, it does not function in this record.
+* Projects: You can select which projects (If you have more than one project) this field should display for.
+* You may also see fields about publishing, if you have the publishing module.  See: 
+Click save once you have completed the appropriate fields.
+
+__Editing a field__
+
+Choose the field you wish to edit in the centre column, and select it.  In the right-hand column, you will see the field information appear (you may need to scroll up if you have a long list of fields.)  Double click on the non-bold text, where it says 'Field: _name_.  From here follow the process as above to edit and save the field.
+
+__Types of fields__
+
+* Select box - Select a single option from the list of options.
+* Multi-select box - to select several options, hold the ctrl key down and select the options you need.
+* Plain text area - enter a short amount of free text.
+* Plain text box - enter longer amounts of free text
+* Rich text - enter free text here and add formatting as desired.
+* Check box - click on the box to fill in a tick
+* Radio button - click on the button to indicate a yes
+* Dates - Dates are entered in UK format by selecting from the day/month/year select boxes or using the popup calendar.
+* Date/Times - Dates and times are entered on a 24-hour clock from select boxes.
+* Date of birth selector - Dates are entered in UK format by selecting from the day/month/year select boxes and a current age is displayed
```

Comment:
Date of birth fields also give you options in {{group}}s to do things like 'people aged between 18 and 24', and to look at anniversaries etc.  So 'date of birth' fields can also be useful for things like 'Date of membership'.